apps/oxlint/src/lint.rs

```diff
@@ -483,18 +501,20 @@ mod test {
         assert_eq!(result.number_of_errors, 0);
     }
 
+    // Previously, this test would pass and the unmatched rule would be ignored, but now we report that
+    // there was unmatched rule, because the typescript plugin has been disabled and we are trying to configure it.
     #[test]
+    #[should_panic(
```

Comment:
Instead of testing for a panic that's coming out of the `test` function, shall we add another `test_failure` function instead.